### ISSUE TITLE
AdyenCardInternal module missing in carthage build

### ISSUE THREAD

**Issue by ffittschen** [Adyen/adyen-ios#83]
Hi,
we want to integrate Adyen into our project using carthage, but we always get the following build error on our CI:

```
Missing required module 'AdyenCardInternal'
Testing cancelled because the build failed.
```

Somehow, the header files and the modulemap contained in the `AdyenCard/Utilities/AdyenCSE` folder are not copied into any of the frameworks built by carthage, therefore the AdyenCardInternal module is not being found.

How could this issue be resolved?

## Steps to reproduce
1. Add Adyen to project using carthage
2. Check in the _Carthage/Build_ folder into version control
3. Build on any other machine that only has the _Carthage/Build_ folder, but not the _Carthage/Checkouts_ folder

**Comment by diegomarcon** [comment]
Hi @ffittschen 

Thanks for the report. 
In which version of Adyen you see this behaviour? 

Cheers

**Comment by ffittschen** [comment]
Thanks for the quick response @diegomarcon 

We are using the latest version, i.e., `3.0.0`.

Here are excerpts from our Cartfile and Cartfile.resolved:

<details open>

<summary>Cartfile</summary>

```
github "adyen/adyen-ios"
```

</details>


<details open>

<summary>Cartfile.resolved</summary>

```
github "adyen/adyen-3ds2-ios" "0.9.6"
github "adyen/adyen-ios" "3.0.0"
```

</details>

**Comment by diegomarcon** [comment]
Thanks for the info. 

We will investigate and get back to you when we have more information.

**Comment by ffittschen** [comment]
Hi @diegomarcon, are there any updates regarding this issue?

**Comment by diegomarcon** [comment]
Hi @ffittschen, 

We are changing the way we handle internal modules and that should fix your issue. 
I expect this to be on next release (next week).

Cheers,

**Comment by ffittschen** [comment]
Thank you, this is great news!

**Comment by diegomarcon** [comment]
Hi @ffittschen,

We just released [3.1.0](https://github.com/Adyen/adyen-ios/releases/tag/3.1.0) that contains the change I previously mentioned. 

I'm closing this issue as it should be fixed by this release. 
Please let me know if you run into any other issue!

Cheers,

**Comment by ffittschen** [comment]
Confirming the fix. Thank you for the support!
